docs/running-on-yarn.md

```diff
@@ -99,6 +101,8 @@ to the same log file).
 
 If you need a reference to the proper location to put log files in the YARN so that YARN can properly display and aggregate them, use `spark.yarn.app.container.log.dir` in your `log4j.properties`. For example, `log4j.appender.file_appender.File=${spark.yarn.app.container.log.dir}/spark.log`. For streaming applications, configuring `RollingFileAppender` and setting file location to YARN's log directory will avoid disk overflow caused by large log files, and logs can be accessed using YARN's log utility.
 
+To use a custom metrics.properties for the application master and executors, just update the `$SPARK_CONF_DIR/metrics.properties` file, it will automatically uploaded with other configurations, so you don't need to specify it manually with --files.
```

Comment:
Remove "just"; "file, it will automatically" -> "file. It will automatically be". Back-tick `--files`